site/en/docs/devtools/settings/preferences/index.md

```diff
@@ -0,0 +1,686 @@
+
+### Appearance
+
+This section lists options that customize DevTools appearance.
+
+{% Details %}
+
+{% DetailsSummary %}
+**Themes** {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/VPpFJAIWgNSaTmnYrqNP.svg", alt="Drop-down.", width="24", height="24" %} sets a color theme for DevTools UI.
+{% endDetailsSummary %}
+
+<fieldset>
+   <legend>Themes:</legend>
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/m0URI9jodUrMxioYEZ11.gif", alt="Changing DevTools theme from system preference to dark to light.", width="400", height="400", style="float:right;", class="screenshot" %}
+
+- System preference
+- Light
+- Dark
+
+</fieldset>
+{% endDetails %}
+
+{% Details %}
+
+{% DetailsSummary %}
+**Panel layout** {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/VPpFJAIWgNSaTmnYrqNP.svg", alt="Drop-down.", width="24", height="24" %} arranges panes in panels.
+{% endDetailsSummary %}
+
+Affects **Elements** > **Styles** and sister tabs, and the **Sources** > **Debugger** pane. The **auto** option makes the layout depend on DevTools width.
+
+<fieldset>
+   <legend>Panel layout:</legend>
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/RO1dKsa37xhMhqCKRD6j.gif", alt="Changing the Element's panel layout from horizontal to vertical.", width="400", height="400", style="float:right;", class="screenshot" %}
+
+- horizontal
+- vertical
+- auto
+
+</fieldset>
+{% endDetails %}
+
+{% Details %}
+
+{% DetailsSummary %}
+**Color format** {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/VPpFJAIWgNSaTmnYrqNP.svg", alt="Drop-down.", width="24", height="24" %} sets the format of CSS color values in **Elements** > **Styles**.
+{% endDetailsSummary %}
+
+DevTools automatically converts any valid color value to the chosen format.
+
+<fieldset>
+   <legend>Color format:</legend>
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/EbCbudgcqUV01eWiGEWP.gif", alt="Changing color formats in the Styles pane.", width="350", height="350", style="float:right;", class="screenshot" %}
+
+- As authored
+- HEX: #dac0de
+- RGB: rgb(128 255 255)
+- HSL: hsl(300deg 80% 90%)
+
+</fieldset>
+{% endDetails %}
+
+{% Details %}
+
+{% DetailsSummary %}
+**Language** {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/VPpFJAIWgNSaTmnYrqNP.svg", alt="Drop-down.", width="24", height="24" %} sets the locale for DevTools UI.
+{% endDetailsSummary %}
+
+To apply this setting, reload DevTools.
+
+<fieldset>
+   <legend>Language:</legend>
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/UTiCVB0hCtDhrUyHumc6.png", alt="The Settings panel in Chinese.", width="400", height="400", style="float:right;" %}
+
+- Browser UI language
+- One of locale options, in this example, Chinese
+
+</fieldset>
+{% endDetails %}
+
+{% Details %}
+
+{% DetailsSummary %}
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/hmp8j3HiLMCcqPArD9yt.svg", alt="Checkbox.", width="22", height="22" %} **Enable <kbd>Ctrl/Cmd</kbd> + <kbd>0</kbd>-<kbd>9</kbd> shortcut to switch panels** lets you open panels using the keyboard.
```

Comment:
From this point onward, each setting has a checkbox show at the start. This is not the case for the prior settings. Does this has any indication?

Reply:
I think after reading the whole docs, I guess that mean the setting is enabled by default. If this is the case, we should include an explanation in the beginning of the docs on what these icons indicate.

Reply:
No, not enabled by default. The previous options were drop-down lists (˅), not checkboxes. I'll add an explanation about icons.